packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.spec.tsx

```diff
@@ -531,6 +532,26 @@ describe('HttpOperation', () => {
       expect(screen.queryByText('some_property')).not.toBeInTheDocument();
     });
   });
+
+  describe('Visibility', () => {
+    it('should hide TryIt', async () => {
+      render(
+        <TryItProvider hideTryIt>
```

Comment:
I don't see any reason why we shouldn't simply pass `hideTryIt` directly to HttpOperation, just like `docsOnly`.

Reply:
We could do that, though we need that property on API level, which would mean that we need to add it to `APIWithSidebarLayout` too and pass it to Docs there. My argument was that layout component shouldn't need to know about this property. Do you think it's fine to do that? Unless you mean to still use context but read it on Docs level, not HttpOperation?

Reply:
Quite honestly, I don't see a problem.

API has `hideTryIt` property, so for me it's 100% okay for `ApiWithSidebarLayout` to have that property as well. :shrug:

Reply:
I know that `context` makes passing that data easier, but I believe we overrely on Providers a bit. Rendering a component in isolation requires us to use 4-5 providers, which is a bit crazy.

Thus I believe we should avoid Providers where it's possible, and here it is absolutely possible in my opinion.

Reply:
Well, if you're fine with that then I guess we can have it this way. My idea was that we will use it later in new Provider, which we might not be able to avoid, but we can think of it when we decide to refactor it. For now I'll keep it simple and just pass the property then.

Reply:
I've updated it so it doesn't use Provider. 
I've also added hiding TryIt for stacked layout, but then I had to use local context. Adding that prop to Group and Item was to painful for me :wink:. In this layout it didn't make much sense hiding just TryIt component. We dont have there examples (no reason for that) and samples are bound to TryIt data so I just hid whole tab for that case.